src/components/App.tsx

```diff
@@ -855,8 +831,11 @@ class App extends React.Component<ExcalidrawProps, AppState> {
 
     history.record(this.state, this.scene.getElementsIncludingDeleted());
 
-    if (this.props.onChange) {
-      this.props.onChange(this.scene.getElementsIncludingDeleted(), this.state);
+    if (this.props.onChangeEmitter) {
+      this.props.onChangeEmitter.emit(
+        this.scene.getElementsIncludingDeleted(),
+        this.state,
+      );
```

Comment:
ignore why this is an emitter for now

Reply:
We should have this as a callback as it was before.